### ISSUE TITLE
Dict unpacking with two unpacked arguments fails

### ISSUE THREAD

**Issue by daviskirk** [python/mypy#2086]
This is valid (and pretty usefull) syntax in python 3.5:

``` python
dict(**{'a': 1}, **{'b': 2})
```

But mypy gives you an error:

```
error: Parse error before **
```

Note that this is different than #704 as it only describes a single unpacked argument.  This is present in Mypy 0.4.4 as well as the git version as of today.

**Comment by daviskirk** [comment]
I just noticed that running mypy with `--fast-parser` does not throw the same error.  Is this the intended behaviour?

**Comment by Michael0x2a** [comment]
It is, yeah -- the fast parser is generally more feature-complete and less buggy then the current parser. For example, `async`/`await` is supported only by the fast parser, there are [some bugs](https://github.com/python/mypy/issues?q=is%3Aopen+is%3Aissue+label%3A%22New+Parser%22) that appear only when using the old parser, etc.

The plan is to switch to using the fast parser by default and get rid of the current parser once we figure out how to get the fast parser ([which is largely written in C](https://github.com/dropbox/typed_ast)) to compile on Windows.

**Comment by gvanrossum** [comment]
This can be closed once the fast parser is the default.

**Comment by gvanrossum** [comment]
I'm going to close this as a won't fix because the new parser will solve this and we're not going to put any work in the old parser/lexer.
